faith_presentation/src/commonMain/kotlin/net/thechance/mena/faith/presentation/designSystem/theme/QuranTheme.kt

```diff
@@ -0,0 +1,27 @@
+package net.thechance.mena.faith.presentation.designSystem.theme
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.runtime.ReadOnlyComposable
+import net.thechance.mena.designsystem.presentation.theme.theme.MenaTheme
+import net.thechance.mena.faith.presentation.designSystem.typography.LocalQuranTypography
+import net.thechance.mena.faith.presentation.designSystem.typography.QuranTextStyle
+import net.thechance.mena.faith.presentation.designSystem.typography.createQuranTypography
+import net.thechance.mena.designsystem.presentation.theme.typography.Typography as MenaTypography
+
+@Composable
+fun QuranTheme(content: @Composable () -> Unit) {
+    MenaTheme {
+        val quranTypography: QuranTextStyle = createQuranTypography()
```

Comment:
After applying the other comments the code will be like that
```suggestion
        val fontFamily = QuranTypography.getFontFamily()
        val quranTypography = remember { QuranTypography.create(fontFamily = fontFamily) }
```

Reply:
done